### DISCUSSION TITLE
BarStackHorizontal for single column

### DISCUSSION THREAD

**Discussion by lukkna** [airbnb/visx#1234] [Q&A]
sorry, stupid question

**Comment by williaster** [comment]
Hey @lukkna ! thanks for checking out `visx`. There are no stupid questions 🙂

I don't think we have any examples of this, but I'm wondering if something like the following would suffice? (modified [sandbox](https://codesandbox.io/s/frosty-gould-8lfn3?file=/Example.tsx), simply filtered to one datum).

![image](https://user-images.githubusercontent.com/4496521/121597404-888c5200-c9f5-11eb-8c14-77113c2a6e3d.png)

You could omit the axes if you don't need them. If you could provide an example of what you are shooting for perhaps I can provide better suggestions.
